### PR TITLE
Alarm functionality - not using bluetooth connectivity, just notification and snooze 

### PR DESCRIPTION
**Pull request by hannahfolkertsma**

### CODE REVIEW

**Comment by hannahfolkertsma** [comment]
[Alarm Functionality (iOS Watch)](https://trello.com/c/eFDUvfti/93-alarm-functionality-ios-watch)

File: src/watch/hearRingiOSWatchOS/hearRingiOSWatchOS WatchKit Extension/NotificationController.swift [hunk 1]
```diff
@@ -11,13 +11,16 @@ import UserNotifications
...
 
+    let alarms = Alarm()
     override var body: NotificationView {
         return NotificationView()
     }
 
     override func willActivate() {
         // This method is called when watch view controller is about to be visible to user
         super.willActivate()
+        //alarms.getAlarms()
```
Comment (hannahfolkertsma) [inline src/watch/hearRingiOSWatchOS/hearRingiOSWatchOS WatchKit Extension/NotificationController.swift:22]: i realized i forgot to take these comments out, but i can remove them in my next alarm PR. if whoever reviews this has a problem with this ill get rid of em though

Comment (hannahfolkertsma) [reply src/watch/hearRingiOSWatchOS/hearRingiOSWatchOS WatchKit Extension/NotificationController.swift:22]: oh nvm i will take em out i dont need any of these changes anymore

**Review by ashleyreneepalmer — APPROVED** [review]
Well done!! Works beautifully :D

**Pull request merged**